docs/user/security/authentication/index.asciidoc

```diff
@@ -138,7 +138,7 @@ xpack.security.authc.saml.maxRedirectURLSize: 1kb
 ==== OpenID Connect Single Sign-On
 
 Similar to SAML, authentication with OpenID Connect allows users to log in to {kib} using an OpenID Connect Provider such as Google, or Okta. OpenID Connect
-should also be configured in {es}, see {xpack-ref}/saml-guide.html[Configuring OpenID Connect Single-Sign-On on the Elastic Stack] for more details.
+should also be configured in {es}, see {xpack-ref}/oidc-guide.html[Configuring OpenID Connect Single-Sign-On on the Elastic Stack] for more details.
```

Comment:
This page has moved from the Stack Overview to the Elasticsearch Reference:

```suggestion
should also be configured in {es}. For more details, see {ref}/oidc-guide.html[Configuring single sign-on to the {stack} using OpenID Connect].
```

Reply:
Thanks!